docker-compose.yml

```diff
@@ -21,7 +21,7 @@ services:
     image: mongo:latest
     ports:
       - 27019:27017
-    restart: unless-stopped
+    # restart: unless-stopped
```

Comment:
please un command this line for starting this container when docker run on you machine

Reply:
Done